website/docs/developer/development-guide/code/versioning-releases.md

```diff
@@ -100,6 +97,13 @@ base branch.
 
 :::
 
+## Publish Documentation
+
+In order to release the ORCA documentation, use the [`Release ORCA Documentation` plan under ORCA](https://ci.earthdata.nasa.gov/browse/ORCA-ROD) and then follow the steps in `Creating a Bamboo deployment plan branch` section above.
+Make sure to replace your bamboo github variables before running the pipeline.
+
+Note that this plan can only target the `develop` and `master` branches. Make sure that this step is run after your release branch has been merged back into develop and master.

```

Comment:
You could also run the plan against the release branch and that will also work.

Reply:
Might reword a little bit here that..
Either use the release branch or use the master branch once release branch has been merged into master.

Reply:
Added tested section.